### PR TITLE
IP-196 Done

### PR DESCRIPTION
**Pull request by domatix**
You need create the custom field. You should put in Label = "Pago" and in Table = "Invoice".

### CODE REVIEW

File: application/modules/invoices/views/view.php [hunk 1]
```diff
@@ -288,14 +288,20 @@ class="form-control input-sm"
...
 
-            <?php foreach ($custom_fields as $custom_field) { ?>
-                <label><?php echo $custom_field->custom_field_label; ?></label>
-                <input type="text" class="form-control"
-                       name="custom[<?php echo $custom_field->custom_field_column; ?>]"
-                       id="<?php echo $custom_field->custom_field_column; ?>"
-                       value="<?php echo form_prep($this->mdl_invoices->form_value('custom[' . $custom_field->custom_field_column . ']')); ?>"
-                    <?php if ($invoice->is_read_only == 1) { echo 'disabled="disabled"';} ?>>
-            <?php } ?>
+             <?php if($custom_field->custom_field_label=="Pago"){?>
```
Comment (Kovah) [inline application/modules/invoices/views/view.php]: Did you used a custom field to store the payment method in there?

**Comment by Kovah** [comment]
Needs a complete rework... :/

**Comment by cz-vilda** [comment]
I understand this commit well.

You have to write payment method directly to invoice, at Czech republic. Its told by law. For this, I use custom filed "payment method" (at Invoice table) and I change invoice template to use it.

At my point of view is not necessary to change whole system, but custom fields is ready to be used for this. I think, that make possibility custom fields "mandatory"/"not mandatory" (just as option at custom fields settings) and not just text fileds (but also selects) is right way to do it.

But its about discus...

(I have no time to do it now, but after i finish ToDo lists, I do it).

**Comment by Kovah** [comment]
I would rather add a new field to the invoice table than force the user to create a custom field with a specific name to display the the payment method on the invoice.

**Pull request closed without merging**